Userland/Utilities/less.cpp

```diff
@@ -216,8 +217,18 @@ class Pager {
         }
     }
 
+    size_t calculate_digits_count(size_t line_numbers)
+    {
+        return line_numbers > 0 ? (int)log10(line_numbers) + 1 : 1;
```

Comment:
Use `static_cast` instead of a C-style cast - also why cast to `int` when we want `size_t`?